src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v2/common.kt

```diff
@@ -1009,3 +1066,48 @@ fun isRefundableCheckRequired(tx: BaseTransaction): Boolean =
       setOf(TransactionStatusDto.AUTHORIZATION_COMPLETED, TransactionStatusDto.CLOSURE_REQUESTED)
         .contains(tx.status) && !wasAuthorizationDenied(tx)
   }
+
+fun <T> postponeRefundProcessingRequest(
```

Comment:
This doesn't actually `postpone` the request, what about:
```suggestion
fun <T> computeRefundProcessingRequestDelay(
```
?